pkgs/tools/networking/bacnet-stack/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "bacnet-stack";
+  version = "2019-04-14";
```

When not using release numbers, packages should be versioned with unstable-2019-04-14

```suggestion
  name = "bacnet-stack";
  version = "unstable-2019-04-14";
```